### PR TITLE
ES-38 : move ES integration code to PLF code base

### PR DESCRIPTION
**Pull request by thomasdelhomenie**
Add required dependencies for Elasticsearch integration.

### CODE REVIEW

**Comment by mtrabelsi** [comment]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -1077,6 +1095,21 @@
         <artifactId>xpp3</artifactId>
         <version>${xpp3.version}</version>
       </dependency>
+      <dependency>
```
Comment (mgreau) [inline pom.xml]: Like for maven properties names, all dependencies are sorted by groupId name in alphabetical order.

Comment (thomasdelhomenie) [inline pom.xml]: Done.

**Comment by mgreau** [comment]
LGTM

**Pull request merged**